oct/pages/models/registration_page.py

```diff
@@ -0,0 +1,54 @@
+from oct.pages.base.page import BasePage
+from oct.pages.base.elements import CheckBox, RadioButton, RadioButtonGroup, Form, Block, InputField
+
+
+class NewsLetter(RadioButtonGroup):
+    contains = {
+        "yes": {"locator": ("XPATH", '//*[@id="content"]/form/fieldset[3]/div/div/label[1]/input'),
+                "class": RadioButton},
+        "no": {"locator": ("XPATH", '//*[@id="content"]/form/fieldset[3]/div/div/label[2]/input'),
+               "class": RadioButton}
+    }
+
+
+class YourPassword(Block):
+    contains = {
+        "password": {"locator": ("ID", "input-password"), "class": InputField},
+        "confirm": {"locator": ("ID", "input-confirm"), "class": InputField}
+    }
+
+
+class PersonalDetails(Block):
+    contains = {
+        "first_name": {"locator": ("ID", "input-firstname"), "class": InputField},
+        "last_name": {"locator": ("ID", "input-lastname"), "class": InputField},
+        "email": {"locator": ("ID", "input-email"), "class": InputField},
+        "telephone": {"locator": ("ID", "input-telephone"), "class": InputField}
+    }
+
+
+class RegistrationForm(Form):
+    contains = {
+        "personal": {"locator": ("XPATH", '//*[@id="account"]'), "class": PersonalDetails},
+        "password": {"locator": ("XPATH", '//*[@id="content"]/form/fieldset[2]'), "class": YourPassword},
+        "newsletter": {"locator": ("XPATH", '//*[@id="content"]/form/fieldset[3]'), "class": NewsLetter},
+        'policy': {"locator": ("XPATH", '//*[@id="content"]/form/div/div/input[1]'), "class": CheckBox}
```

Comment:
Take into account that a locator should be relative to the parent webelement (you are searching an input in the form, you don't need to include a path to the form in the locator, just a path inside of the form to this input otherwise nothing will be found). cc @YuriiShp